62.CAD/common.hlsl

```diff
@@ -115,31 +135,15 @@ struct PSInput
     [[vk::location(1)]] nointerpolation uint4 data1 : COLOR1;
     [[vk::location(2)]] nointerpolation float4 data2 : COLOR2;
     [[vk::location(3)]] nointerpolation float4 data3 : COLOR3;
+    [[vk::location(4)]] nointerpolation float4 data4 : COLOR4;
+    // Data segments that need interpolation, mostly for hatches
+    [[vk::location(5)]] float4 interp_data5 : COLOR5;
```

Comment:
`Default Initialize PS Input` in vertex shader, I already do it for the other members